bindings/protocols/modbus/index.html

```diff
@@ -308,6 +314,10 @@ <h2>Mappings</h2>
         model</a>. </p>
         <section id="default-mappings">
             <h3>Default mappings</h3>
+            <p>
+                The following table list the default mappings between the protocol specific concepts and the WoT concepts. Notice operations that are not
+                listed in the table are not supported by this binding template. For example, since the ModBus protocol is a request-response protocol, the
```

Comment:
```suggestion
                listed in the table are not supported by this binding template. For example, since the ModBus protocol is a request-response based protocol, the
```